app/views/users/index.html.erb

```diff
@@ -1,4 +1,11 @@
 <div id="home-card">
+
+<h1>Welcome to my Blog 🥳</h1>
+
+<%= link_to "/posts/new" do %>
+    <button>Create a Post</button>
+<% end %>
```

Comment:
- [ ] [optional] @JakariaJishan great work adding a link to the page to add a new post, however, I would recommend you move it to the usershow page, preferably below the posts.